components/organisms/Contributors/contributors.tsx

```diff
@@ -35,8 +38,22 @@ const Contributors = ({ repositories }: ContributorProps): JSX.Element => {
   const router = useRouter();
   const limit = router.query.limit as string;
   const topic = router.query.pageId as string;
+  const { range = 30 } = router.query;
 
   const { data, meta, setPage, isError, isLoading } = useContributors(Number(limit ?? 10), repositories);
```

Comment:
Using this as the data for allContributors count... subject to change based on response from slack